test/unit/mdc-auto-init/mdc-auto-init.test.js

```diff
@@ -95,3 +96,47 @@ test('#register warns when registered key is being overridden', () => {
 
   td.verify(warn(contains('(mdc-auto-init) Overriding registration')));
 });
+
+test('#emit dispatches a custom event with supplied data', () => {
```

Comment:
Consider rewording '#emit dispatches a custom event with supplied data' > '#mdcAutoInit dispatches a *MDCAutoInit:End* event when initialization of all components is complete'